src/realm/group_shared.hpp

```diff
@@ -526,6 +526,19 @@ class SharedGroup {
     // Release pinned version (not thread safe)
     void unpin_version(VersionID version);
 
+    // Delete the realm file and coordination files. The lock file is not deleted,
```

Comment:
Perhaps be really explicit that the lock file is in the same directory  as the file?